### PR TITLE
[CHI-390] Added / Fixed backstop tests for several components

### PR DESCRIPTION
**Pull request by janibolkvadze**
@jllr @mattnickles @SergioQCL @dani-cl-madrid

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/344/ for more details.

File: test/chi/components/picker.pug [hunk 1]
```diff
@@ -8,66 +8,75 @@ h3 Base
...
-          input(type="radio", class="a-picker", name='box', id=`rd-${type}-${size}`)
-          label(for=`rd-${type}-${size}`)
-            .m-picker__content--start
-              label.-text--bold
+
+h3 Sizes
+.test-sizes
+  .example.-mb--3(style='width: 700px')
+    .-p--3
+      each size in sizes
```
Comment (jllr) [inline test/chi/components/picker.pug]: @janibolkvadze sizes examples are not the same as in picker page sizes examples

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/356/ for more details.

**Comment by mattnickles** [comment]
@janibolkvadze can you resolve conflicts?

**Comment by SergioQCL** [comment]
@janibolkvadze Please, solve conflicts

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/391/ for more details.

**Pull request merged**